extensions/components/com_redcore/admin/views/webservice/tmpl/default.php

```diff
@@ -14,150 +14,12 @@
 JHtml::_('behavior.keepalive');
 JHtml::_('rbootstrap.tooltip');
 JHtml::_('rjquery.chosen', 'select');
+JHtml::script(JUri::root() . '/media/redcore/js/redcore.js');
```

Comment:
Never mind, I didn't realize that the mini.js files are saved in the extension directory. :smile: